WalletWasabi.Fluent/ViewModels/Wallets/Send/SendViewModel.cs

```diff
@@ -242,10 +242,9 @@ private bool TryParseUrl(string? text)
 		if (AddressStringParser.TryParse(text, _wallet.Network, out BitcoinUrlBuilder? url))
 		{
 			result = true;
-			SmartLabel label = url.Label;
-
-			if (!label.IsEmpty)
+			if (url.Label is { })
 			{
+				SmartLabel label = url.Label;
 				_transactionInfo.UserLabels = new SmartLabel(label.Labels);
 			}
```

Comment:
How about this:

```suggestion
			if (url.Label is { } label)
			{
				SmartLabel smartLabel = label;
				_transactionInfo.UserLabels = new SmartLabel(smartLabel.Labels);
			}
```

Reply:
It could be `_transactionInfo.UserLabels = new SmartLabel(label.Labels)` because when you do this `url.Label is { } label` it is already a new reference

Reply:
Does it make any significant difference?
Or is it just for better readability?

Reply:
> It could be `_transactionInfo.UserLabels = new SmartLabel(label.Labels)` because when you do this `url.Label is { } label` then `label` is already a new reference

It can't because `label` is a string.

```
Does it make any significant difference?
Or is it just for better readability?
```

Yes for better readability. Feel free to ignore.

Reply:
Let's go with the one above.